docs/conceptual/runtime/index.md

```diff
@@ -2,6 +2,32 @@
 title: Runtime Overview
 ---
 
-What is the Substrate Runtime?
+## Substrate Runtime
 
+The runtime is the most critical component in your blockchain. 
+It contains the business logic that will define your blockchain's behavior,
+a.k.a. the state transition function.
```

Comment:
Subjective stylistic advice: "Most critical" sounds funny because the blockchain will not work at all without block authoring, gossip, etc. I like your second sentence better. How about:

The runtime contains the business logic that will define your blockchain's behavior,
a.k.a. the state transition function.